screens/StartupScreen.js

```diff
@@ -9,11 +9,40 @@ import { useDispatch } from 'react-redux';
 
 import Colors from '../constants/Colors';
 import * as authActions from '../store/actions/auth';
+import * as Permissions from 'expo-permissions';
+import Constants from 'expo-constants';
+import useAsyncEffect from '../hooks/useAsyncEffect';
 
 const StartupScreen = (props) => {
   const isMountedRef = useRef(null);
   const dispatch = useDispatch();
 
+  useAsyncEffect(async () => {
+    if (Constants.isDevice) {
+      const { status: existingStatus } = await Permissions.getAsync(Permissions.NOTIFICATIONS);
+      let finalStatus = existingStatus;
+      if (existingStatus !== 'granted') {
+        const { status } = await Permissions.askAsync(Permissions.NOTIFICATIONS);
+        finalStatus = status;
+      }
+      if (finalStatus !== 'granted') {
+        alert('Failed to get push token for push notification!');
```

Comment:
I'll go through and change all English notifications to Swedish at a later stage